src/Component/Nav/HeaderNav.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const HeaderNav = () => {
+  const [inputBg, setInputBg] = useState(false);
+
+  const onInputFocused = () => {
+    setInputBg(!inputBg);
+  };
```

Comment:
```suggestion
  const onInputFocused = () => setInputBg(!inputBg)
```
1. 한 줄로 써줄 수 있습니다
2. 하지만 애초에 그냥 setInputBg를 곧바로 써주면 될 것 같습니다 (얘도 함수입니다 - 속닥 -)

Reply:
속닥 속닥